flask_resty/jwt.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, jwk_set, *args, **kwargs):
         self.jwk_set = jwk_set
 
     def decode(self, jwt, **kwargs):
-        unverified_header = self.get_unverified_header(jwt)
+        unverified_header = jwt_lib.get_unverified_header(jwt)
```

Comment:
This got moved to a new `PyJWS` object which can be accessed directly via the `jwt` module